Mathlib/Geometry/Manifold/MFDeriv/NormedSpace.lean

```diff
@@ -27,17 +27,17 @@ variable {𝕜 : Type*} [NontriviallyNormedField 𝕜]
   {E' : Type*}
   [NormedAddCommGroup E'] [NormedSpace 𝕜 E'] {H' : Type*} [TopologicalSpace H']
   {I' : ModelWithCorners 𝕜 E' H'} {M' : Type*} [TopologicalSpace M'] [ChartedSpace H' M']
-  [SmoothManifoldWithCorners I' M']
+  [IsManifold I' n M']
   -- declare a smooth manifold `N` over the pair `(F, G)`.
```

Comment:
Please update this comment; also the one below.

Reply:
(And perhaps search for similar examples in mathlib: I only looked at the diff so far, not searched mathlib if there were further patterns.)